jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletChannel.java

```diff
@@ -480,13 +480,15 @@ public boolean handle()
 
                     case DISPATCH:
                     {
-                        dispatch(_requestDispatchable);
+                        reopen();
```

Comment:
Why do we need this `reopen()`?

Reply:
perhaps not.  We were doing it before in dispatch method, but I'll try without and see what happens....

Reply:
it wasn't necessary

Reply:
oh yes it was.  It is needed for multiple requests on the same connection.